core/src/test/scala/org/apache/spark/deploy/worker/DriverRunnerTest.scala

```diff
@@ -45,6 +52,18 @@ class DriverRunnerTest extends SparkFunSuite {
     (processBuilder, process)
   }
 
+  private def createTestableDriverRunner(processBuilder: ProcessBuilderLike,
```

Comment:
nit: wrong alignment for multi-line params.

See https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Indentation